modin/experimental/pandas/test/test_io_exp.py

```diff
@@ -111,6 +111,11 @@ def test_read_csv_empty_frame(self):
 
         df_equals(modin_df, pandas_df)
 
+    def test_read_csv_without_glob(self):
+        with pytest.warns(UserWarning, match=r"Shell-style wildcard"):
+            with pytest.raises(FileNotFoundError):
+                pd.read_csv_glob("s3://nyc-tlc/trip data/yellow_tripdata_2020-")
```

Comment:
Does it warn quickly? Probably, would be better to use a simple small data from our bucket: `s3://modin-datasets/testing/multiple_csv/`

Reply:
`s3://nyc-tlc/trip data/yellow_tripdata_2020-` - is not correct URL. There is no actual data reading.

Reply:
Ok